model/common_structures.smithy

```diff
@@ -59,3 +59,15 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+structure CertificatesDetail {
+    issuer_dn: String,
+    subject_dn: String,
+    san: String,
+    not_before: String,
+    not_after: String
+}
+
+list CertificatesList {
+    member: CertificatesDetail
+}
```

Comment:
This file will grown extremely large if we put every structure here. It's only meant for structure to be reused like the results of different types of search. Can we move these into the structures.smithy file of the respective operation?